Tests/Marketplace/approved_tags.json

```diff
@@ -36,6 +36,7 @@
 		"Courses of Action",
 		"Unit 42",
 		"Best Practices",
-		"New"
+		"New",
```

Comment:
I think you can remove New tag. It is auto added as needed and shouldn't be added to packs. @yaakovi FYI